src/server/node_services/nodes_monitor.js

```diff
@@ -1386,44 +1382,10 @@ class NodesMonitor extends EventEmitter {
         }
     }
 
-    async _test_store_perf(item) {
```

Comment:
instead of this function that stored the results to DB, add a log message in `blcok_store_base.test_store_perf` that prints the results to log1